### DISCUSSION TITLE
Config Errors following adding object detection.

### DISCUSSION THREAD

**Discussion by ghost** [blakeblackshear/frigate#8755] [Ask A Question] [answered]
Hey everyone, I have recently been doing some edits to my config and I am not getting this error when trying to save it any ideas?

    Config Error:
    
    Traceback (most recent call last):
      File "/opt/frigate/frigate/http.py", line 761, in config_save
        new_yaml = FrigateConfig.parse_raw(new_config)
      File "/opt/frigate/frigate/config.py", line 1083, in parse_raw
        return cls.parse_obj(config)
      File "pydantic/main.py", line 526, in pydantic.main.BaseModel.parse_obj
      File "pydantic/main.py", line 341, in pydantic.main.BaseModel.__init__
    pydantic.error_wrappers.ValidationError: 1 validation error for FrigateConfig
    events
      extra fields not permitted (type=value_error.extra)

  
-----------------------------------------
I have added my configuration below:

    mqtt:
      enabled: false
    
    cameras:
      FrontDoor:
        enabled: true
        ffmpeg:
          inputs:
            - path: rtsp://USERNAME:PASSWORD@192.168.1.95/ISAPI/Streaming/Channels/101
              roles:
                - record
            - path: rtsp://USERNAME:PASSWORD@192.168.1.95/ISAPI/Streaming/Channels/102
              roles:
                - detect
                - rtmp
    
      Driveway:
        enabled: true
        ffmpeg:
          inputs:
            - path: rtsp://USERNAME:PASSWORD@192.168.1.96/ISAPI/Streaming/Channels/101
              roles:
                - record
            - path: rtsp://USERNAME:PASSWORD@192.168.1.96/ISAPI/Streaming/Channels/102
              roles:
                - detect
                - rtmp
    
      Greenhouse:
        enabled: true
        ffmpeg:
          inputs:
            - path: rtsp://USERNAME:PASSWORD@192.168.1.97/ISAPI/Streaming/Channels/101
              roles:
                - record
            - path: rtsp://USERNAME:PASSWORD@192.168.1.97/ISAPI/Streaming/Channels/102
              roles:
                - detect
                - rtmp
    
      Cabin:
        enabled: true
        ffmpeg:
          inputs:
            - path: rtsp://USERNAME:PASSWORD@1080@192.168.1.98/ISAPI/Streaming/Channels/101
              roles:
                - record
            - path: rtsp://USERNAME:PASSWORD@192.168.1.98/ISAPI/Streaming/Channels/102
              roles:
                - detect
                - rtmp
    
    birdseye:
      enabled: True
      mode: continuous
    
    detect:
      width: 1280
      height: 720
      fps: 24
    
    objects:
      track:
        - person
        - dog
        - cat
        - plate
    
    snapshots:
      enabled: True
    
    record:
      enabled: True
      retain:
        days: 5
    
    events:
      retain:
        default: 10

**Comment by NickM-27** [comment, marked as answer]
events config is a property of record so it should be indented
